srcs/commands/Join.cpp

```diff
@@ -11,65 +12,113 @@
 
 Join::Join() {}
 Join::~Join() {}
-Join::Join(const Join& other) : ICommand(), _channelsLst(other._channelsLst) {}
+Join::Join(const Join& other) : ICommand(), _chans(other._chans) {}
 Join& Join::operator=(const Join& other)
 {
     if (this != &other) {
-        _channelsLst = other._channelsLst;
+        _chans = other._chans;
     }
     return *this;
 }
 
-Join::Join(const std::vector<std::string>& channelsLst) : _channelsLst(channelsLst) {}
-
 /**
- * @brief check if there is no params for command Join, and parse the params
- * and format this params to a vector of string <channel><space><key>
+ * @brief recreate the channels list. Each name of channel are mapped with their corresponding key.
```

Comment:
if longer than one line, it can be 
```suggestion
 * @brief recreate the channels list.
 * @details each channel name is mapped with its corresponding key.
```